tests/test_upstream_versions.py

```diff
@@ -1694,3 +1696,63 @@ def test_github_version_prefix(url, version, version_prefix, tmpdir):
         assert gh.version_prefix is None
     else:
         assert gh.version_prefix == version_prefix
+
+
+@pytest.mark.parametrize(
+    "url, feedstock_version",
+    [
+        ("https://github.com/spglib/spglib/archive/v2.3.0.tar.gz", "2.3.0"),
+    ],
+)
+@flaky
+def test_github_releases(tmpdir, url, feedstock_version):
+    meta_yaml = LazyJson(os.path.join(tmpdir, "cf-scripts-test.json"))
+    with meta_yaml as _meta_yaml:
+        _meta_yaml.update(
+            {
+                "version": feedstock_version,
+                "url": url,
+            }
+        )
+
+    ghr = GithubReleases()
+    url = ghr.get_url(meta_yaml)
+    assert VersionOrder(ghr.get_version(url)) > VersionOrder(feedstock_version)
+
+
+@pytest.mark.parametrize(
+    "url, feedstock_version, latest_release, latest_tag",
+    [
+        (
+            "https://github.com/spglib/spglib/archive/v2.3.0.tar.gz",
+            "2.3.0",
+            "2.4.0",
+            "2.5.0",
+        ),
+    ],
+)
+@flaky
+def test_github_tag_not_released(
```

Comment:
Since this has passed on CI now, I'd say we remove this. The moment upstream releases 2.5.0 this will start failing and it's just going to be noise.

WDYT?

Reply:
yep sounds good to me!